### ISSUE TITLE
[Jetformbuilder] Search in select

### ISSUE THREAD

**Issue by nreljed** [CrocoBlock/suggestions#5305]
search and visual hierarchy inside select
ideal for selecting from a large number of choices
like country or a very long list imported by glossary

![select search](https://user-images.githubusercontent.com/82481592/166724563-46d2c962-94fc-49a6-82d4-c89f2069663a.gif)

**Comment by edgBlw** [comment]
Yeah, that would be great. I know you have such a similar addon in the jetformbuilder account. But the styling then is just too bad. also, i think, you can't see all options from the select field with that addon. You only get the available ones that are matching the search query which is kind of bad for the user, because you can't get an impression which fields and how many of them are available for you.
![Bildschirmfoto 2022-05-06 um 16 58 24](https://user-images.githubusercontent.com/101653721/167159723-8de49929-7beb-4b45-8c5d-a3327c5d7d08.jpg)

**Comment by mehdimoradi1500** [comment]
+1

**Comment by satomiao** [comment]
+1 up

**Comment by git-temak** [comment]
+1
